spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -107,7 +107,9 @@ The following Annotations exist, see below for common fields:
 |===
 |Annotation | Applies to |  Description | Default Unit
 
-|@Counted | M, C, T | Denotes a counter, which counts the invocations of the annotated object. | MetricUnits.NONE
+|@Counted | M, C, T | DEPRECATED Denotes a counter, which counts the invocations of the annotated object. This can be the number of parallel or absolute invocations. To be replaced by @HitCounted and @ParallelCounted | MetricUnits.NONE
+|@HitCounted | M, C, T | Denotes a counter, which counts the invocations of the annotated object. Each invocation increases the counter by 1.| MetricUnits.NONE
+|@ParallelCounted | M, C, T | Denotes a counter, which counts the parallel invocations of the annotated object. | MetricUnits.NONE
```

Comment:
We discussed this on the call. I think this metrics should be specified as Gauge not counter.